### DISCUSSION TITLE
I'm trying to run this program but I'm having a little difficulty

### DISCUSSION THREAD

**Discussion by Timmy-Chien** [forestagostinelli/deepxube#4] [Q&A] [answered]
We would like to run this program.
How should I configure the 'State Class' in Environment.md? If possible, could you provide an example?
Really appreciate it, thanks!!!

**Comment by forestagostinelli** [comment, marked as answer]
Yes, you can find examples of implementations of State here:

https://github.com/forestagostinelli/deepxube/blob/7eac59426ddf3b7fbe168c5cefca8651dd68171e/deepxube/environments/cube3.py#L79
https://github.com/forestagostinelli/deepxube/blob/7eac59426ddf3b7fbe168c5cefca8651dd68171e/deepxube/environments/n_puzzle.py#L22
https://github.com/forestagostinelli/deepxube/blob/7eac59426ddf3b7fbe168c5cefca8651dd68171e/deepxube/environments/sokoban.py#L49
